Symfony/CS/Resources/phar-stub.php

```diff
@@ -20,6 +20,10 @@
     exit(1);
 }
 
+set_error_handler(function ($errno, $errstr, $errfile, $errline) {
+    throw new ErrorException($errstr, $errno, 1, $errfile, $errline);
```

Comment:
Imo you should pass the `$errno` to the third parameter (`severity`).
http://php.net/manual/en/class.errorexception.php#errorexception.examples (example 1)